frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -62,6 +62,14 @@ export const ReportStreamHeader = () => {
                 >
                     <span>{FeatureName.DAILY_DATA}</span>
                 </USNavLink>
+                // Will be needed when all STLT pages are developed
```

Comment:
Can we leave out any commented code until we need it? Any commented code in a `render` appears in the DOM.

Reply:
I was under the impression, for at least the production build, comments are stripped from the final bundle. Is this not the case?

Reply:
I haven't seen any evidence of that, but I can run a production build locally to text. The only solution I've seen recommended is: https://www.npmjs.com/package/webpack-comment-remover-loader

In either case, we shouldn't have comments within our `render` functions. I think comments within the logic beforehand is fine for complicated business logic.

Reply:
I see now. To be clear (and I agree), there should not be comments within JSX blocks (which are rendered). Easy fix, just move the comments to above the `itemsMenu.push(` (within the logic side).